[#318] Snapshot comments fetching via API

filip-kopecky opened the pull request:
Comments for snapshotted terms are now fetched via specific API call. 

Resolves: #318

filip-kopecky committed:
[#318] Snapshot comments fetching via API

Added the option to send optional date parameters to loadTermComments method.

ledsoft commented on src/action/AsyncCommentActions.ts:
```diff
@@ -15,13 +15,21 @@ import Comment, {
+    const reqParams: any = {};
+    reqParams.namespace = termIri.namespace;
+    if (dateTo) reqParams.to = dateTo;
+    if (dateFrom) reqParams.from = dateFrom;
```
I think you can actually skip the `if`s, as params with undefined/nullish value will not be added to the request (see `paramsSerializer` in Ajax).

filip-kopecky replied:
You are right, I totally missed that. Added commit which removes the `if`s

filip-kopecky committed:
[#318] Removed unnecessary if statements

Removed parameter checking. It is not passed if value is null, therefore if statement is not required

ledsoft reviewed (approved):
LGTM
